proto/viam/app/cloudslam/v1/cloud_slam.proto

```diff
@@ -24,9 +24,11 @@ message StartMappingSessionRequest {
   // Version to use for slam, defaults stable
   string slam_version = 2;
   string map_name = 3;
-  string org_id = 4;
+  string organization_id = 4;
   string location_id = 5;
   string robot_id = 6;
+  // Version to use for viam, defaults stable
+  string viam_server_version = 7;
```

Comment:
I don't see this in the scope. Should this be added?

Reply:
I have just added it, along with the other variables that needed to be added to the StartMappingSessionRequest

Reply:
Thank you. Please make sure to send out a scope update.